plugdj/base.py

```diff
@@ -18,12 +19,28 @@ def _post(self, path, return_req=False, **kwargs):
         req = self._session.post(self.to_url(path), **kwargs)
         if return_req:
             return req
+        req.raise_for_status()
```

Comment:
This would count as a bug fix in its own right. Thanks for catching it. Possible to separate it out?